src/main/java/org/zalando/failsafeactuator/FailsafeAutoConfiguration.java

```diff
@@ -0,0 +1,26 @@
+package org.zalando.failsafeactuator;
```

Comment:
[Can you move this to package `autoconfigure` please? ](https://github.com/spring-projects/spring-boot/wiki/Building-On-Spring-Boot)

Reply:
Based on Spring's own usage, should the packages be like this?

- `org.zalando.actuate.autoconfigure.failsafe`:
  - `FailsafeAutoConfiguration`
- `org.zalando.actuate.failsafe`
  - `CircuitBreakerEndpoint`

Comparing this to Spring's own health endpoint:

- `org.springframework.boot.actuate.autoconfigure.health`:
  - `HealthEndpointAutoConfiguration`
- `org.springframework.boot.actuate.health`
  - `HealthEndpoint`